.changeset/green-trees-wait.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/create-app': patch
+---
+
+imports `useSearch` hook from new `@backstage/plugin-search-react` package.
+
+To upgrade existing Apps change the imports in the following two files:
+
+`components/search/SearchModal.tsx`
```

Comment:
Not this one right? Just the search page. Gotta add the `@backstage/plugin-search-react` dependency to the app too

Reply:
Bonus points for making all path references start from the repository root.
`packages/app/src/components/search/SearchModal.tsx` in this case I guess? :)

Reply:
@Rugvip both, see changes to the SearchModal here https://github.com/backstage/backstage/pull/10842/files#diff-506348f267c82faeef8233367f4e9e644fd0623eea34db9f48a5ac20f9e9f1e2

Reply:
@Rugvip both, see changes to the SearchModal here https://github.com/backstage/backstage/pull/10842/files#diff-506348f267c82faeef8233367f4e9e644fd0623eea34db9f48a5ac20f9e9f1e2

Reply:
oh never mind, mixed up app and default app. Fixing!

Reply:
I mean as an instruction to the user. Right now they'd end up with a lint error because of the missing dependency